### DISCUSSION TITLE
License concerns

### DISCUSSION THREAD

**Discussion by potrepka** [acids-ircam/RAVE#195] [Q&A] [answered]
The license you have chosen is CC BY-NC.

Does this license prohibit musicians from using your software to create music for commercial purposes?

**Comment by caillonantoine** [comment, marked as answer]
Hi ! No it does not ! The sounds you produce using your pretrained models are yours, and you can use them for concert / music production etc... This license is here to prevent rave from being packaged as part of a paid product :)

**Reply by potrepka** [reply]
Thank you for the clarification!
